zmq/__init__.py

```diff
@@ -38,10 +38,26 @@ def _load_libzmq():
 
 def get_includes():
     """Return a list of directories to include for linking against pyzmq with cython."""
+    from os.path import join, dirname, abspath, pardir, exists
+    base = dirname(__file__)
+    parent = abspath(join(base, pardir))
+    includes = [ parent ] + [ join(parent, base, subdir) for subdir in ('utils',) ]
+    if exists(pjoin(parent, base, 'include')):
+        includes.append(pjoin(parent, base, 'include'))
+    return includes
+    
+def get_library_dirs():
+    """Get the library directory for linking to the bundled copy of libzmq."""
     from os.path import join, dirname, abspath, pardir
     base = dirname(__file__)
     parent = abspath(join(base, pardir))
-    return [ parent ] + [ join(parent, base, subdir) for subdir in ('utils',) ]
+    global _libzmq
+    try:
+        _libzmq
```

Comment:
Thinking about this more: This is probably the wrong way to detect bundled libzmq at runtime. What is the right way?